knative/knative-serving-install/base/cluster-role.yaml

```diff
@@ -118,6 +134,7 @@ rules:
   - admissionregistration.k8s.io
   resources:
   - mutatingwebhookconfigurations
+  - validatingwebhookconfigurations
```

Comment:
The files for the webhook configurations seem to be missing.

Reply:
the webhook configurations used to be generated, in 0.11.1 they moved to manifests.